api-service/database/alerts_test.go

```diff
@@ -448,3 +448,12 @@ func (m *MongodbSuite) TestUpdateAlertsStatus() {
 		clean()
 	}
 }
+
+func (m *MongodbSuite) TestRemoveAlertsNODATA() {
+	defer m.db.Client.Database(m.dbname).Collection(alertsCollection).DeleteMany(context.TODO(), bson.M{})
+
+	filter := dto.AlertsFilter{OtherInfo: map[string]interface{}{"hostname": "foobar"}}
+
+	err := m.db.RemoveAlertsNODATA(filter)
+	require.NoError(m.T(), err)
```

Comment:
This tests check that method do not return any error, that's ok!

But can we tests even that it do remove NO_DATA alert correctly?
Or that e.g. if there are two NO_DATA alerts for two different hosts, it delete only the correct one?

Thanks